scripts/integtest.sh

```diff
@@ -95,18 +95,19 @@ fi
 USERNAME=`echo $CREDENTIAL | awk -F ':' '{print $1}'`
 PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
 
-OPENSEARCH_HOME=`ps -ef | grep -o "[o]pensearch.path.home=\S\+" | cut -d= -f2- | head -n1`
+OS="`uname`"
 
-curl -SL https://raw.githubusercontent.com/opensearch-project/sql/main/integ-test/src/test/resources/datasource/datasources.json -o "$OPENSEARCH_HOME"/datasources.json
-
-yes | $OPENSEARCH_HOME/bin/opensearch-keystore add-file plugins.query.federation.datasources.config $OPENSEARCH_HOME/datasources.json
-
-if [ $SECURITY_ENABLED == "true" ] 
+if [ $OS != "WindowsNT" ]
```

Comment:
GHA on Windows runs powershell scripts by default (you can override this, but install bash first)
You can omit this `if`.

Reply:
Windows instances that we use in our build comes with bash pre-installed.
`Didn't get you on omitting if clause.`